src/lib/slider/slider.ts

```diff
@@ -226,6 +235,15 @@ export class MdSlider implements ControlValueAccessor {
   set vertical(value: any) { this._vertical = coerceBooleanProperty(value); }
   private _vertical = false;
 
+  /** The value to be used for display purposes. */
+  get displayValue(): string|number {
+    if (this._roundLabelTo && this.value % 1 !== 0) {
+      return this.value.toFixed(this._roundLabelTo);
```

Comment:
I think there's a couple cases this misses:
.999 to 2 places
.899 to 2 places
.001 to 2 places